generators/app/templates/src/index.js

```diff
@@ -0,0 +1,51 @@
+export default {
+    // Multiple browsers support
+    isMultiBrowser: <%= isMultiBrowser %>,
+
+
+    // Required - must be implemented
+    // Browser control
+    async openBrowser (/* id, browserName, startPage */) {
+        throw new Error('Not implemented!');
+    },
+
+    async closeBrowser (/* id, pageInfo */) {
```

Comment:
if `browserInfo` turned out to be `browserName`... I suspect that `pageInfo` isn't something we expect here also. What is it? `pageUrl`?